addon/components/frost-checkbox.js

```diff
@@ -1,34 +1,50 @@
+/**
+ * Component definition for the frost-checkbox component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  run,
-  set,
-  typeOf
-} = Ember
+const {Component, get, isEmpty, run, set, typeOf} = Ember
 import PropTypeMixin, {PropTypes} from 'ember-prop-types'
+
 import layout from '../templates/components/frost-checkbox'
 
 export default Component.extend(PropTypeMixin, {
-  // == Properties ============================================================
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
 
-  classNames: ['frost-checkbox'],
   classNameBindings: ['size'],
+  classNames: ['frost-checkbox'],
   layout,
 
+  // == PropTypes =============================================================
+
+  /**
+   * Properties for this component. Options are expected to be (potentially)
+   * passed in to the component. State properties are *not* expected to be
+   * passed in/overwritten.
+   */
```

Comment:
Actually, it's fine to have this from the generator, I just wouldn't expect it to be kept afterward